### DISCUSSION TITLE
Faciliter les actualisations de fiche membre [difficultés Github]

### DISCUSSION THREAD

**Discussion by fannyblanc** [betagouv/cap#8] [Besoin]
### Besoin

```bash
Deux soucis identitifés avec github : 
* On demande aux utilisateur.ices de merger leur fiche membre / la fiche membre d'un membre de leur équipe mais cela n'a rien de facile. Beaucoup de personnes ne savent pas faire. 
* Github est en anglais : pas inclusif pour les personnes qui ne parlent pas cette langue.
```

### Solutions envisageables

```bash
1. Voir si on peut proposer une version en français de github (je ne sais pas si ça existe)
2. Trouver un système pour que les gens n'aient pas besoin de passer par github 
3. Y-a t-il une alternative à Github ?
```

### Informations complémentaires

- [X] J'avais déjà entendu ce besoin au sein de la communauté beta.gouv
- [ ] Je me porte volontaire pour participer à la mise en place de la future solution choisie

**Comment by revolunet** [comment]
Que penserais-tu d'utiliser l'espace membre pour gérer ces infos (et celles des SE) ?

**Reply by AmelAlili** [reply]
Les modifications peuvent déjà se faire via l'espace membre mais il faut toujours aller sur github pour merger (mais tout le monde ne comprend pas qu'en validant sur l'espace membre, ça ne valide pas vraiment).

**Reply by LaurentM83** [reply]
+1 pour poursuivre la simplification mais déjà un grand merci pour tout ce qui a été fait, l'édition des fiches produits et membre est beaucoup plus simple maintenant qu'il y a 2 ans!

**Reply by revolunet** [reply]
Ok, donc ce serait l'option 2 "Trouver un système pour que les gens n'aient pas besoin de passer par github"

**Comment by revolunet** [comment]
WIP : https://github.com/betagouv/espace-membre-next/pull/197

**Comment by revolunet** [comment]
On peut clôturer 🎉
